presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -307,6 +311,19 @@ private void processPage(Page page)
             for (Aggregator aggregator : aggregators) {
                 aggregator.processPage(groupIds, page);
             }
+
+            recordHashCollisions();
```

Comment:
Have you benchmarked the performance impact of this? It looks like `estimateNumberOfHashCollisions` could be significant if the pages are small